deepomatic/api/http_helper.py

```diff
@@ -75,9 +75,9 @@ def __init__(self, app_id=None, api_key=None, verify_ssl=None,
             app_id = os.getenv('DEEPOMATIC_APP_ID')
         if api_key is None:
             api_key = os.getenv('DEEPOMATIC_API_KEY')
-        if app_id is None or api_key is None:
-            raise DeepomaticException("Please specify 'app_id' and 'api_key' either by passing those values to the client"
-                                      " or by defining the DEEPOMATIC_APP_ID and DEEPOMATIC_API_KEY environment variables.")
+        if api_key is None:
+            raise DeepomaticException("Please specify 'api_key' either by passing those values to the client"
+                                      " or by defining the DEEPOMATIC_API_KEY environment variables.")
```

Comment:
`s/variables/variable/`

Reply:
done in done in https://github.com/Deepomatic/deepomatic-client-python/pull/69